core/assets/bundles/bundle_nl.properties

```diff
@@ -194,68 +194,68 @@ save.delete.confirm = Weet je zeker dat je dit opgeslagen spel wilt verwijderen?
-quit.confirm = Are you sure you want to quit?
-quit.confirm.tutorial = Are you sure you know what you're doing?\nThe tutorial can be re-taken in[accent] Settings->Game->Re-Take Tutorial.[]
-loading = [accent]Loading...
-reloading = [accent]Reloading Mods...
-saving = [accent]Saving...
-cancelbuilding = [accent][[{0}][] to clear plan
-selectschematic = [accent][[{0}][] to select+copy
-pausebuilding = [accent][[{0}][] to pause building
-resumebuilding = [scarlet][[{0}][] to resume building
-wave = [accent]Wave {0}
-wave.waiting = [LIGHT_GRAY]Wave in {0}
-wave.waveInProgress = [LIGHT_GRAY]Wave in progress
-waiting = [LIGHT_GRAY]Waiting...
-waiting.players = Waiting for players...
-wave.enemies = [LIGHT_GRAY]{0} Enemies Remaining
-wave.enemy = [LIGHT_GRAY]{0} Enemy Remaining
-loadimage = Load Image
-saveimage = Save Image
-unknown = Unknown
+copylink = Kopieer Link
+back = Terug
+data.export = Exporteer Data
+data.import = Importeer Data
+data.exported = Data geëxporteerd.
+data.invalid = Dit is geen geldige spel data.
+data.import.confirm = Door externe gegevens te importeren, worden [scarlet] al[] je huidige spelgegevens gewist.\n[accent]Dit kan niet ongedaan worden![]\n\nZodra de gegevens zijn geïmporteerd, wordt je spel onmiddellijk afgesloten.
+classic.export = Exporteer klassieke data
+classic.export.text = [accent]Mindustry[] heeft zojuist een grote update gehad.\nKlassieke (v3.5 build 40) opgeslagen spellen of mappen zijn gedetecteerd. Wilt u deze opgeslagen bestanden exporteren naar de thuismap van uw telefoon, voor gebruik in de Mindustry Classic-app?
+quit.confirm = Weet je zeker dat je wilt stoppen?
+quit.confirm.tutorial = Weet je zeker dat je weet wat je doet?\nDe tutorial kan opnieuw gedaan worden in [accent] Instellingen->Spel->Tutorial herkansen.[]
+loading = [accent]Laden...
+reloading = [accent]Mods herladen...
+saving = [accent]Opslaan...
+cancelbuilding = [accent][[{0}][] plan weghalen
+selectschematic = [accent][[{0}][] selecteren + kopieren
+pausebuilding = [accent][[{0}][] bouwen pauzeren
+resumebuilding = [scarlet][[{0}][] bouwen hervatten
+wave = [accent]Golf {0}
+wave.waiting = [LIGHT_GRAY]Golf over {0}
+wave.waveInProgress = [LIGHT_GRAY]Golf bezig
+waiting = [LIGHT_GRAY]Wachten...
+waiting.players = Wachten voor spelers...
+wave.enemies = [LIGHT_GRAY]{0} Vijanden over
+wave.enemy = [LIGHT_GRAY]{0} Vijand over
+loadimage = Afbeelding laden
+saveimage = Afbeelding opslaan
+unknown = Onbekend
 custom = Custom
-builtin = Built-In
-map.delete.confirm = Are you sure you want to delete this map? This action cannot be undone!
-map.random = [accent]Random Map
-map.nospawn = This map does not have any cores for the player to spawn in! Add a[ROYAL] blue[] core to this map in the editor.
+builtin = Ingebouwd
+map.delete.confirm = Weet je zeker dat je deze map wilt verwijderen? Deze actie kan niet ongedaan worden!
```

Comment:
Grammatical error:
Where's the verb? (See comment 2)

Reply:
thnx